tests/testutils.py

```diff
@@ -32,3 +33,22 @@ def create_and_process_template_response(template, context=None, use_middleware=
     else:
         response.render()
     return response.content.decode("utf-8")
+
+
+def print_nodes(nodes: List[Node], indent=0) -> None:
```

Comment:
Added a convenience func to print out nodeslist, which I used a lot during debugging